modules/distribution/src/repository/resources/conf/default.json

```diff
@@ -186,5 +186,9 @@
       "type": "httpbasic",
       "primary": false
     }
-  }
+  },
+
```

Comment:
Unnecessary new line

Reply:
These files were moved to the component level and refactored.